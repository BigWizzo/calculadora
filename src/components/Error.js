import React from 'react';

const Error = () => (
  <div>
    404: Page not Found
  </div>
);

export default Error;
