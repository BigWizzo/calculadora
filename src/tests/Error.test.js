import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Error from '../components/Error';

test('should render the Home Page', () => {
  const component = renderer.create(
    <BrowserRouter>
      <Error />
    </BrowserRouter>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
