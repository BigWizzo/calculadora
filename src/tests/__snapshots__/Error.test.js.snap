// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Home Page 1`] = `
<div>
  404: Page not Found
</div>
`;
